build.assets/makefiles/base/systemd/system.conf

```diff
@@ -0,0 +1,2 @@
+[Manager]
+DefaultLimitNOFILE=655350:655350
```

Comment:
Where does the colon come from - can you link to the docs on this syntax?

Reply:
From systemd [docs](https://www.freedesktop.org/software/systemd/man/systemd.exec.html):

> Resource limits may be specified in two formats: either as single value to set a specific soft and hard limit to the same value, or as colon-separated pair soft:hard to set both limits individually (e.g. "LimitAS=4G:16G").

Only in testing it appeared that specifying the single value didn't really override the soft limit so we set it explicitly.